pkg/auth/manager/memory/memory.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2018-2021 CERN
```

Comment:
We're in 2022 already since a while :smile:

Reply:
yep, tooling problem